libraries/lib-time-and-pitch/StaffPadTimeAndPitch.cpp

```diff
@@ -1,13 +1,12 @@
 #include "StaffPadTimeAndPitch.h"
-#include "AudioContainer.h"
 
 #include <algorithm>
 #include <cassert>
 #include <memory>
 
 namespace
 {
-constexpr auto maxBlockSize = 512;
+constexpr auto maxBlockSize = 128;
```

Comment:
Oops, that was for experimenting, I should revert that.

- [ ] @saintmatthieu revert

Reply:
So we are not making it configurable for now? That's fine, but I don't think that fixing it will take any time